interop-testing/src/main/java/io/grpc/testing/integration/AbstractInteropTest.java

```diff
@@ -966,6 +972,8 @@ protected void assertX500SubjectDn(String tlsInfo) {
 
     assertEquals(1, certificates.size());
     assertEquals(tlsInfo, x509cert.getSubjectDN().toString());
+    assertEquals(serverCallCapture.get().attributes().get(Grpc.TRANSPORT_ATTR_SSL_SESSION),
```

Comment:
nm. I see. They are duped within the transport. Let's have one key a copy of the other, as I suggested above. That should be simpler.